modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LinkTag.java

```diff
@@ -0,0 +1,347 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib;
+
+import com.liferay.frontend.taglib.clay.internal.servlet.taglib.BaseContainerTag;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.util.Validator;
+import com.liferay.taglib.util.TagResourceBundleUtil;
+
+import java.util.Set;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.JspWriter;
+
+/**
+ * @author Chema Balsas
+ */
+public class LinkTag extends BaseContainerTag {
+
+	@Override
+	public int doStartTag() throws JspException {
+		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
+
+		setContainerElement("a");
+
+		if (Validator.isNotNull(_ariaLabel)) {
+			setDynamicAttribute(StringPool.BLANK, "aria-label", _ariaLabel);
+		}
+
+		if (Validator.isNotNull(_href)) {
+			setDynamicAttribute(StringPool.BLANK, "href", _href);
+		}
+
+		if (Validator.isNotNull(_target)) {
+			setDynamicAttribute(StringPool.BLANK, "rel", "noreferrer noopener");
```

Comment:
Maybe this should be possible to override. Or failing that, maybe we should only set `noreferrer noopener` for some links (ie. if `_target` is not `_self`).

Reply:
Yeah, I thought about it.. just copied from https://github.com/liferay/clay/blob/master/packages/clay-link/src/index.tsx#L87-L89

Will dig in later to see how we came to this conclusion...

Reply:
Actually, I was expecting that a value in `rel` passed to the tag as a dynamic attribute would be set over this based on the order of those methods executing... but it might so happen that the order is reversed, so we'd need to check here for the existence of the `rel` dynamic attribute.